### ISSUE TITLE
Default Country not working when using the min js file

### ISSUE THREAD

**Issue by mohammedkhan** [Bluefieldscom/intl-tel-input#184]
Hello, me again. I found the auto selection of country was not working when using the min js file, works fine with the non-min file.

**Comment by jackocnr** [comment]
Hmm I just tried and I can't reproduce this. Please tell me what version of the plugin you're using, as well as all of the initialisation options you are using, and please explain exactly what happens - what do you mean by "not working"? Do you see any flag? Or a grey square instead? Is there a JavaScript error? Thanks.

**Comment by mohammedkhan** [comment]
Sorry I realised I wasn't being very descriptive. I'm using version 5.8, if I attempt to add `defaultCountry: "auto"` to the `intlTelInput` function for my input I lose the country selection drop down, i.e. it looks as though there's nothing loaded. Chrome tells me `Uncaught TypeError: undefined is not a function`, but as it's a min file the whole thing is on a single line and Developer Tools can't tell me the specific area of code that is the cause of the problem. Here is the snippet of code I'm using:
```html
    <script src="intl-tel-input-master/build/js/intlTelInput.min.js"></script>
    <script>
      var $j = jQuery.noConflict();
      $j("#tel_day").intlTelInput({
    	  defaultCountry: "auto",
    	  utilsScript: "intl-tel-input-master/lib/libphonenumber/build/utils.js",
    	  preferredCountries: ["gb"]
    	});
    </script>
```
Simply changing `intlTelInput.min.js` to `intlTelInput.js` fixes the problem.

**Comment by jackocnr** [comment]
A ha! I think this was an incompatibility with jQuery noConflict. I think I've fixed it in v5.8.1 - could you try it and let me know?

**Comment by mohammedkhan** [comment]
Yep working now, thanks :+1:
